### PR TITLE
Fix peer dep missing: eslint-plugin-import

### PR DESCRIPTION
**Pull request by dplusic**
Fixed by updating eslint-config-airbnb package.

### CODE REVIEW

File: app/main.development.js [hunk 1]
```diff
@@ -30,7 +30,7 @@ const installExtensions = async () => {
       'REDUX_DEVTOOLS'
     ];
     const forceDownload = !!process.env.UPGRADE_EXTENSIONS;
-    for (const name of extensions) {
+    for (const name of extensions) { // eslint-disable-line
```
Comment (amilajack) [inline app/main.development.js]: Why is this line being eslint-disable'd?

Comment (dplusic) [inline app/main.development.js]: Airbnb restrict 'for ... of' statement in their style guide.
https://github.com/airbnb/javascript/pull/1134

**Comment by amilajack** [comment]
Any idea why the build is failing?

**Comment by dplusic** [comment]
Resolve peer dependencies problem.

**Pull request merged**